### DISCUSSION TITLE
Try to add the connect to a device feature from spotify

### DISCUSSION THREAD

**Discussion by Zippu-4249** [pear-devs/pear-desktop#2614] [Ideas]
# Try to add the connect to a device feature from spotify
![image](https://github.com/user-attachments/assets/b1fd4fe1-cfec-48c2-97a1-57bbc3dc60de)

**Comment by pauserra-dev** [comment]
@Zippu-4249 Yeah it would be very useful. Tbh it's the only feature that I miss from Spotify.
